dv/uvm/riscv_dv_extension/ibex_log_to_trace_csv.py

```diff
@@ -0,0 +1,53 @@
+"""
+Copyright lowRISC contributors.
+Licensed under the Apache License, Version 2.0, see LICENSE for details.
+SPDX-License-Identifier: Apache-2.0
```

Comment:
Please use `#` for the license header to avoid having this header in the documentation block.

Reply:
Done